mlflow/sklearn/__init__.py

```diff
@@ -607,6 +652,24 @@ def _save_model(sk_model, output_path, serialization_format):
             import cloudpickle
 
             _dump_model(cloudpickle, sk_model, out)
+        elif serialization_format == SERIALIZATION_FORMAT_SKOPS:
+            try:
+                import skops.io
+            except ImportError as e:
+                raise MlflowException(
+                    message=(
+                        "Failed to import skops. Make sure skops is installed by running "
+                        "`pip install skops` for instance."
+                    ),
+                    error_code=INTERNAL_ERROR,
+                ) from e
+
+            dumped = skops.io.dumps(sk_model, compresslevel=9)
+            untrusted_types = skops.io.get_untrusted_types(data=dumped)
+            out.write(dumped)
+            if untrusted_types:
+                _logger.info(f"Detected untrusted types for skops format: {untrusted_types}")
```

Comment:
automatic saving `untrusted_types` (so loading model loads these `untrusted_types` automatically) is unsafe.